dependencies.yaml

```diff
@@ -252,10 +252,10 @@ dependencies:
           - pytest>=6.2.4
           - tifffile>=2022.7.28
           - pooch>=1.6.0  # needed to download scikit-image sample data
-          - matplotlib
+          - matplotlib-base
```

Comment:
`matplotlib-base` doesn't exist on PyPI so it won't work for `requirements` or `pyproject` outputs. You'll need to list `matplotlib-base` in a `conda` output and `matplotlib` for `[requirements, pyproject]`.
```suggestion
```